app/__init__.py

```diff
@@ -25,6 +27,8 @@ def create_app(config_class, database_uri=None):
     app = Flask(__name__, static_url_path="/assets")
     app.config.from_object(config_class())
 
+    inspect.getmembers(config_class())
```

Comment:
I think it will be better to just create the class object once above:

```
config = config_class()
inspect.getmembers(config)
app.config.from_object(config)
```

Reply:
updated changes as requested above